spec/models/roster_spec.rb

```diff
@@ -147,4 +147,21 @@
       end
     end
   end
+
+  describe 'user_options' do
+    let(:roster) { create :roster }
+    let!(:admins) { [roster_admin(roster)] }
+    let!(:non_admins) { [roster_user(roster)] }
+    let(:call) { roster.user_options }
+
+    it 'has admins in the "Admins"' do
+      expect(call.fetch('Admins'))
+        .to contain_exactly(*admins.map { |a| [a.full_name, a.id] })
```

Comment:
Wouldn't `expect(call.fetch('Admins')).to eq(admins.map { |a| [a.full_name, a.id] })` be sufficient?

Reply:
or maybe `match_array` would be the more appropriate matcher?

Reply:
What's wrong with `contain_exactly`? I use it all the time so I'm curious

Reply:
There are two different improvement paths @jforgue proposed here. In the first, he was pointing out that, in this test, there is only one admin, and a one-element array can only be in one order, so `eq` would work here. However, I prefer to explicitly say "in no particular order", I think.

In the second, he suggests `match_array` which would get rid of the need for the splat.  That is, these two expectations are the same:

```ruby
expect(a_thing).to contain_exactly(:a, :b, :c)
expect(a_thing).to match_array([:a, :b, :c])
```